Tests/SentryTests/SentryKSCrashReportConverterTests.m

```diff
@@ -34,7 +34,7 @@ - (void)testConvertReport
     XCTAssertNotNil(event);
     XCTAssertEqualObjects(
         [NSDate dateWithTimeIntervalSince1970:@(1491210797).integerValue], event.timestamp);
-    XCTAssertEqual(event.debugMeta.count, (unsigned long)256);
+    XCTAssertEqual(event.debugMeta.count, (unsigned long)13);
```

Comment:
`m`: We could assert for the expected 13 debugMeta by name or imageAdress elements here. This would improve the test, I think.